src/SprykerEco/Zed/Payone/Business/Payment/PayoneLogsReceiver.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment;
+
+use Generated\Shared\Transfer\PayonePaymentLogCollectionTransfer;
+use Generated\Shared\Transfer\PayonePaymentLogTransfer;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+class PayoneLogsReceiver implements PayoneLogsReceiverInterface
+{
+    public const LOG_TYPE_API_LOG = 'SpyPaymentPayoneApiLog';
```

Comment:
* Mark as protected if not used outside.

Reply:
++